src/utils/files.js

```diff
@@ -16,11 +19,24 @@ function formatBytes(bytes) {
   return `${bytes} B`;
 }
 
+function formatBytesV2(bytes, decimals = 2) {
```

Comment:
Do we want to keep `formatBytes()`? This one seems better in all cases.
And it would avoid this weird naming